smithy-openapi/src/main/java/software/amazon/smithy/openapi/fromsmithy/OpenApiProtocol.java

```diff
@@ -62,6 +62,12 @@ default void updateDefaultSettings(Model model, OpenApiConfig config) {
      * Creates an operation entry, including the method, URI, and operation
      * object builder.
      *
+     * The returned operation object builder should contain protocol-specific fields of an
+     * OpenAPI Operation object, such as: [parameters, requestBody, responses] & [examples for any of the above,
```

Comment:
Did javadoc yell about this `&`? You might need to use `&amp;` because javadoc is so awesome (lol)

Reply:
It didn't, but I'll still change it. resolved.